src/components/Button/Button.js

```diff
@@ -1,17 +1,48 @@
 import React from 'react';
 import { omit } from 'lodash/fp';
-
-import { sizes } from '../../styles/constants';
+import PropTypes from 'prop-types';
 
 import PlainButton from './components/PlainButton';
 import RegularButton from './components/RegularButton';
-import {
-  BUTTON_PROP_TYPES,
-  BUTTON_DEFAULT_PROPS,
-  REGULAR_BUTTON_ONLY_PROPS
-} from './constants';
+import { sizes } from '../../styles/constants';
 
 const { KILO, MEGA, GIGA } = sizes;
+export const SIZE_PROP_TYPE = PropTypes.oneOf([KILO, MEGA, GIGA]);
+
+const SHARED_PROPS = [
```

Comment:
Hm, why did you need to move these back from the constants file? I see you added all the `propTypes` back anyway, so these could still be in a different file, no?